docs/user-guide/ze-working-with-data-sets.md

```diff
@@ -61,24 +92,55 @@
 
    ![Edit and upload a data set member](../images/ze/ZE-edit-upload.gif)
 
+## Uploading a local file to a data set
+
+1. Expand the **DATA SETS** tree in the **Side Bar**.
+
+2. Click on the **Search** icon next to a profile to search for a pattern that matches the desired data set.
+
+   Search results display under the profile in the **Side Bar**.
+
+3. Right-click on the desired PDS and select the **Upload Member...** option to upload a local file as a member of that data set.
+
+   A file explorer window opens.
+4. In the file explorer, select the desired file and click **Upload File**.
+
+   The selected file is uploaded as a member of the data set and appears as a new member in the **DATA SETS** tree after the upload is complete.
+
+## Comparing data set members
+
+1. Expand the **DATA SETS** tree in the **Side Bar**.
+
+2. Click on the **Search** icon next to a profile to search for a pattern that matches the desired data set.
+
+   Search results display under the profile in the **Side Bar**.
+3. Right-click on the desired member and select the **Select for Compare** option.
+4. Right-click a second member and select the **Compare with Selected** option.
+
+   The first selected member displays in an **Editor** on the left; the second member displays on the right.
+
 ## Preventing merge conflicts
 
-1. Expand **DATA SETS** in the **Side Bar**, and navigate to the member you want to edit.
-2. Edit the document in the **Editor** tab.
-3. Press `Ctrl`+`S` or `Command`+`S` to save the changes.
+1. Expand the **DATA SETS** tree in the **Side Bar**.
+2. Click on the **Search** icon next to a profile to search for a pattern that matches the desired data set.
+
+   Search results display under the profile in the **Side Bar**.
+3. Select the member to open the file in an **Editor**.
+4. Edit the document in the **Editor**.
+5. Press the `Ctrl`+`S` or `Command`+`S` keys to save the changes.
 
    If the original content in your local version no longer matches the same file in the mainframe, a warning message displays advising the user to compare both versions.
-4. If necessary, use the editor tool bar to resolve merge conflicts.
+6. If necessary, use the editor tool bar to resolve merge conflicts.
```

Comment:
I'm not totally sure what this means.

Reply:
It means that the tool bar prompts the user, or highlights the merge conflicts and provides instructions on how the user can resolve conflicts. I'll see if I can make this more clear